util/exercise/src/cmd/configure.rs

```diff
@@ -0,0 +1,306 @@
+use serde_json::{self, Value};
+use std::{
+    fs,
+    io::{stdin, stdout, Write},
+    path::Path,
+};
+use utils;
+use uuid::Uuid;
+
+fn get_user_input(prompt: &str) -> String {
```

Comment:
I may have undestood you incorrectly, but the default value will not help in this case, since in the y/N case I do not explicitly check for the N and in the cases with `is_empty` checks I deal with default values of different types (`String`, `u64`, `Vec<String>`) and I don't want to include excess convertions to the `String`.
If I am missing a point here, could you elaborate more on this, please?